nms/app/views/equipment/GatewayLogs.tsx

```diff
@@ -54,20 +45,15 @@ import {useRefreshingDateRange} from '../../components/AutorefreshCheckbox';
 const MAX_PAGE_ROW_COUNT = 10000;
 const EXPORT_DELIMITER = ',';
 const LOG_COLUMNS = [
-  {
-    title: 'Date',
-    field: 'date',
-    type: 'datetime',
-    width: 200,
-    filtering: false,
-  },
-  {title: 'Service', field: 'service', width: 200},
-  {title: 'Tag', field: 'tag', width: 200},
-  {title: 'Type', field: 'logType', width: 200, filtering: false},
-  {title: 'Output', field: 'output', filtering: false},
+  // eslint-disable-next-line prettier/prettier
```

Comment:
Please don't do that. The eslint prettier rule only checks if everything has been formatted.
If somebody uses prettier directly this comment will be ignored.